app/controllers/samples_controller.rb

```diff
@@ -148,7 +148,7 @@ def index
 
   def index_v2
     # this method is going to replace 'index' once we fully migrate to the
-    # discovery views (old one was kept to avoid breaking the current inteface
+    # discovery views (old one was kept to avoid breaking the current interface
     # without sacrificing speed of development and avoid breaking the current interface)
```

Comment:
nit: since you are fixing comments, can you remove the last "and avoid breaking..." since its repeated?

Reply:
done 👍